auto-discovery/cloud-aws/README.md

```diff
@@ -0,0 +1,237 @@
+export AWS_SESSION_TOKEN="<session token>"
+export AWS_REGION="<region>"
+
+# Create the Kubernetes secret for the service to read the credentials from (use -n to configure another namespace)
+# The names of the secret and the keys can be configured through helm values
+kubectl create secret generic aws-credentials --from-literal=aws-access-key-id=$AWS_ACCESS_KEY_ID --from-literal=aws-secret-access-key=$AWS_SECRET_ACCESS_KEY --from-literal=aws-session-token=$AWS_SESSION_TOKEN
+```
+
+#### Create the rule and queue
+
+This sets up a high throughput FIFO queue, because the AutoDiscovery service assumes the messages will be delivered in order.
+EventBridge does not guarantee in order delivery though, so in rare cases the AutoDiscovery might delete scans too early or keep them around for too long.
+
+```bash
+# Create the queue
+name="secureCodeBox-autodiscovery-events"
+queue_url="$(aws sqs create-queue --queue-name "${name}.fifo" --attributes FifoQueue=true,ReceiveMessageWaitTimeSeconds=20,ContentBasedDeduplication=true,DeduplicationScope=messageGroup,FifoThroughputLimit=perMessageGroupId --tags "SCB-AutoDiscovery=" | jq -r ".QueueUrl")"
+queue_arn="$(aws sqs get-queue-attributes --queue-url $queue_url --attribute-names QueueArn | jq -r ".Attributes.QueueArn")"
+
+# Create the rule
+rule_arn="$(aws events put-rule --name "${name}" --description "Gather events for the secureCodeBox AWS AutoDiscovery" --state ENABLED --tags "Key=SCB-AutoDiscovery,Value=" --event-pattern '{
+  "source": ["aws.ecs"],
+  "detail-type": ["ECS Task State Change", "ECS Container Instance State Change", "ECS Deployment State Change"]
+}' | jq -r ".RuleArn")"
+
+# Allow eventbridge to send messages to the queue
+# Depending on the way you give access to the queue to the AutoDiscovery service, you might also need to give permissions to that
+# The AutoDiscovery requires the sqs:ReceiveMessage and sqs:DeleteMessage permissions
+timestamp="$(date +%s)"
+policy='{
+  "Id": "Policy'"${timestamp}"'",
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Sid": "Stmt'"${timestamp}"'",
+      "Action": "sqs:SendMessage",
+      "Effect": "Allow",
+      "Resource": "'"${queue_arn}"'",
+      "Principal": {
+        "Service": "events.amazonaws.com"
+      },
+      "Condition": {
+        "ArnEquals": {
+          "aws:SourceArn": "'"${rule_arn}"'"
+        }
+      }
+    }
+  ]
+}'
+sqs_policy="$(echo $policy | jq -c -j | jq -R -s)"
+aws sqs set-queue-attributes --queue-url $queue_url --attributes '{"Policy": '"${sqs_policy}"'}'
+
+# Send the messages to the queue, if it shows "FailedEntryCount": 0 it worked
+aws --no-cli-pager events put-targets --rule $name --targets '[{"Id": "Id'"$(uuidgen | tr '[:upper:]' '[:lower:]')"'", "Arn": "'"${queue_arn}"'", "SqsParameters": {"MessageGroupId": "secureCodeBox-AutoDiscovery"}}]'
+
+# Set up variables for later use
+export SQS_QUEUE_URL="${queue_url}"
+```
+
+### In / Excluding Resources from the AutoDiscovery
```

Comment:
```suggestion
### Optional: In / Excluding Resources from the AutoDiscovery
```

Reply:
I added this to the docs resturcture commit that also deals with the reordering.